server/memphis_handlers_consumers.go

```diff
@@ -246,10 +246,13 @@ func (s *Server) createConsumerDirect(c *client, reply string, msg []byte) {
 		IsDeleted:        false,
 		MaxAckTimeMs:     int64(ccr.MaxAckTimeMillis),
 		MaxMsgDeliveries: ccr.MaxMsgDeliveries,
+		OptStartSequence: ccr.OptStartSequence,
+		LastMessages:     ccr.LastMessages,
 	}
 
 	if consumerGroupExist {
-		if newConsumer.MaxAckTimeMs != consumerFromGroup.MaxAckTimeMs || newConsumer.MaxMsgDeliveries != consumerFromGroup.MaxMsgDeliveries {
+
+		if newConsumer.MaxAckTimeMs != consumerFromGroup.MaxAckTimeMs || newConsumer.MaxMsgDeliveries != consumerFromGroup.MaxMsgDeliveries || newConsumer.OptStartSequence != consumerFromGroup.OptStartSequence || newConsumer.LastMessages != consumerFromGroup.LastMessages {
```

Comment:
in this case you are updating the current consumer and you said that we cant do that

Reply:
I did it from sdk side. I am adding it in the broker.